src/test/ui/const-generics/argument_order.rs

```diff
@@ -1,18 +1,24 @@
-#![feature(const_generics)]
-#![allow(incomplete_features)]
+// revisions: full min
+#![cfg_attr(full, feature(const_generics))]
+#![cfg_attr(full, allow(incomplete_features))]
+#![cfg_attr(min, feature(min_const_generics))]
 
 struct Bad<const N: usize, T> {
+    //[min]~^ ERROR type parameters must be declared prior to const parameters
     arr: [u8; { N }],
     another: T,
 }
 
 struct AlsoBad<const N: usize, 'a, T, 'b, const M: usize, U> {
-    //~^ ERROR lifetime parameters must be declared prior
+    //[full]~^ ERROR lifetime parameters must be declared prior
+    //[min]~^^ ERROR lifetime parameters must be declared prior to const parameters
```

Comment:
as `full` and `min` have the same error message, you don't have to duplicate that here iirc
```suggestion
    //~^ ERROR lifetime parameters must be declared prior
```